charts/lakefs/README.md

```diff
@@ -105,19 +112,19 @@ helm upgrade -f my-values.yaml my-lakefs lakefs/lakefs --set kv_upgrade=true
 
 ## Configurations
```

Comment:
Is the change below intentional?

Reply:
Yes - it's a fix of the table formatting